test_server.py

```diff
@@ -0,0 +1,83 @@
+import server
+import socket
+import pytest
+from multiprocessing import Process
+
+ADDR = ('127.0.0.1', 8000)
+_CRLF = b'\r\n'
+
+@pytest.yield_fixture()
+def server_setup():
+    process = Process(target=server.start_server)
+    process.daemon = True
+    process.start()
+    yield process
+
+
+@pytest.fixture()
+def client_setup():
+    client = socket.socket(
+        socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
+    )
+    client.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    client.connect(ADDR)
+    return client
+
+
+def test_response_ok(server_setup):
+    assert '200 OK' in server.response_ok()
```

Comment:
in addition, testing that 200 OK is in the string generated by the method says nothing about whether the string is a valid HTTP response.  For example, this assertion passes:

```python
assert '200 OK' in 'I bought 200 OK Cupid memberships for some reason'
```